packages/core/src/api/parsers/html/parseHTML.test.ts

```diff
@@ -59,6 +59,202 @@ async function parseHTMLAndCompareSnapshots(
 }
 
 describe("Parse HTML", () => {
+  it("Parse BlockNote HTML", async () => {
```

Comment:
Can we use something like `blocksToFullHTML` instead of this hardcoded HTML? It makes it difficult to maintain or understand this testcase. Or is the HTML sth different?

Also, I suppose you're saying these cases weren't handled in other places like copy and paste tests? (I figured we had quite some extensive test cases there)

Reply:
True we can, I'll move the cases to copy/paste tests to make things a bit cleaner.

The main issue I was running into was heading levels being parsed as `null` from BlockNote HTML, and this indeed was not covered by existing copy/paste tests. So to prevent issues in the future the new tests cover parsing all basic blocks from BlockNote HTML (except audio, video, and file blocks, since those are pretty similar to images).